build.gradle.kts

```diff
@@ -10,7 +10,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath ("com.android.tools.build:gradle:4.2.1")
+        classpath ("com.android.tools.build:gradle:4.2.2")
```

Comment:
would this work if instead of `4.2.2` we were using a variable in buildSrc?

Reply:
I think it could work, lets make sure we extract it to see it in action. @mahoganykilo  

Im not sure how it will work with having it split in another file 🤔  ( Dependecies.kt )

Reply:
Yeah got ahead of myself here - even by extending the file matcher to scan the buildSrc folder, it's currently not smart enough to work it out. 

We have two options if we want to continue to use renovate, both involve moving our current deps file:
a) move our dependencies into the top level build.gradle, and allow it share down to the submodules. Slightly less effort but may have an effect on our build times.
b) move our dependencies into a .toml file, maybe less effect on build times but more effort.